### ISSUE TITLE
Awesome!

### ISSUE THREAD

**Issue by FGRibreau** [vipshop/vire#1]
Really nice project!

It will be shared in our next [RedisWeekly](http://redisweekly.com)!

In the mean time did you tried it against [Redsmin](https://www.redsmin.com)? If so, it will be our pleasure to promote Vire as compatible with Redsmin :)

**Comment by deep011** [comment]
Thank you!
Redsmin is a very good project. I like it, and what can i do for Redsmin?

**Comment by FGRibreau** [comment]
Simply connect Vire to Redsmin, check that you can use the UI (I think you will need to implement SCAN (or a least KEYS) to make Redsmin working). And then if you output the same "INFO" output as Redis, you will be real-time and historical monitoring + alerting for free for Vire thanks to Redsmin!

**Comment by deep011** [comment]
Scan command you said include hscan, sscan, zscan or not?

**Comment by FGRibreau** [comment]
Yes I'm talking only about SCAN not HSCAN, SSCAN and so on... :)

Le mardi 31 mai 2016, deep <notifications@github.com> a écrit :

> Scan command you said include hscan, sscan, zscan or not?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vipshop/vire/issues/1#issuecomment-222583260>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAIbQmGasv1cxVMyEJfKmQxAxIhm1-Uuks5qG6a8gaJpZM4IpTIt>
> .
>


-- 
Sent from my iPhone
